src/sentry/plugins/sentry_webhooks/plugin.py

```diff
@@ -13,6 +13,10 @@
 from sentry.http import is_valid_url, safe_urlopen
 from sentry.utils.safe import safe_execute
 
+DESCRIPTION = """
+Trigger outgoing HTTP POST requests from Sentry.
```

Comment:
@jonesphillip I think we should give a more detailed explanation of how you can hook up your own service to Sentry and connect it to alert rules.